static/js/util/currency.js

```diff
@@ -15,3 +16,13 @@ const codesToOptions = R.compose(
 );
 
 export const currencyOptions = codesToOptions(cc.codes());
```

Comment:
I know this isn't part of the current PR, but it looks like there is the possibility of a mismatch between the currencies in the library an the currencies supported by the conversion API.  Is that addressed elsewhere or should we open an issue for it?